src/components/TaskCard/AddTaskCard.jsx

```diff
@@ -15,7 +15,7 @@ const AddTaskCard = ({ isCardCreated, onCreateCard }) => {
     setInputText(event.target.value);
   }
 
-  function handleAddTask() {
+  function onSave() {
```

Comment:
```suggestion
  function handleAddTask() {
```

// see https://react.dev/learn/responding-to-events#naming-event-handler-props